source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnSetRenderer.java

```diff
@@ -73,19 +73,17 @@ public View render(
         columnSetLayout.setFlexWrap(FlexWrap.NOWRAP);
         columnSetLayout.setFlexDirection(FlexDirection.ROW);
 
-        long minHeight = columnSet.GetMinHeight();
-        if (minHeight != 0)
-        {
-            columnSetLayout.setMinimumHeight(Util.dpToPixels(context, (int)minHeight));
-        }
-
         ContainerStyle parentContainerStyle = renderArgs.getContainerStyle();
         ContainerStyle styleForThis = ContainerRenderer.GetLocalContainerStyle(columnSet, parentContainerStyle);
 
         for (int i = 0; i < columnVectorSize; i++)
         {
             Column column = columnVector.get(i);
 
+            if(columnSet.GetMinHeight() > column.GetMinHeight()) {
```

Comment:
>{ [](start = 65, length = 1)

[nit] for style consistency with the rest of this code let's put this brace on the next line